src/test/java/de/mediathekview/mserver/crawler/ndr/parser/NdrFilmDetailDeserializerTest.java

```diff
@@ -0,0 +1,125 @@
+                LocalDateTime.of(2018, 3, 28, 23, 20, 0),
+                Duration.ofSeconds(155)
+            },
+            {
+                "https://www.ndr.de/fernsehen/Sturm-der-Liebe,sendung756612.html",
+                "/ndr/ndr_film_detail4.html",
+                "https://www.ndr.de/fernsehen/sturmderliebe1822-ardjson_image-1afbd693-e72e-406a-abec-7b7f7d0bb66f.json",
+                "Sturm der Liebe",
+                "Sturm der Liebe (2883)",
+                "Werner setzt Christoph mit seinem Wissen unter Druck. In die Enge getrieben, verliert Christoph daraufhin die Beherrschung und es kommt zu einer lebensbedrohlichen Situation.",
+                LocalDateTime.of(2018, 3, 20, 8, 10, 0),
+                Duration.ofMinutes(49).plusSeconds(28)
+            },
+            {
+                "https://www.ndr.de/fernsehen/sendungen/tatort/Blutschuld,sendung751232.html",
+                "/ndr/ndr_film_detail_m3u8.html",
+                "https://www.ndr.de/fernsehen/livestream/livestream217-ardjson_image-5e9560f0-bc96-4d5b-8a92-cddd3f77966f.json",
+                "Tatort",
+                "Blutschuld",
+                "Der Abfallunternehmer Harald Kosen ist in seinem Schlafzimmer erschlagen worden. Die Kommissare Saalfeld und Keppler sind geschockt von dem kaltblütigen Vorgehen des Mörders.",
+                LocalDateTime.of(2018, 3, 20, 22, 0, 0),
+                Duration.ofMinutes(90)
+            }
+        });
+  }
+
+  private final String requestUrl;
+  private final String htmlPage;
+  private final String expectedVideoUrl;
+  private final String expectedTopic;
+  private final String expectedTitle;
+  private final String expectedDescription;
+  private final LocalDateTime expectedTime;
+  private final Duration expectedDuration;
+
+  public NdrFilmDetailDeserializerTest(final String aRequestUrl, final String aHtmlPage, final String aExpectedVideoUrl,
+      final String aExpectedTopic, final String aExpectedTitle, final String aExpectedDescription, final LocalDateTime aExpectedTime,
+      final Duration aExpectedDuration) {
+
+    requestUrl = aRequestUrl;
+    htmlPage = aHtmlPage;
+    expectedVideoUrl = aExpectedVideoUrl;
+    expectedTopic = aExpectedTopic;
+    expectedTitle = aExpectedTitle;
+    expectedDescription = aExpectedDescription;
+    expectedTime = aExpectedTime;
+    expectedDuration = aExpectedDuration;
+  }
+
+  @Test
+  public void deserializeTest() {
+
+    final String htmlContent = FileReader.readFile(htmlPage);
+    final Document document = Jsoup.parse(htmlContent);
+
+    final NdrFilmDeserializer target = new NdrFilmDeserializer();
+    final Optional<FilmInfoDto> actual = target.deserialize(new CrawlerUrlDTO(requestUrl), document);
+
+    assertThat(actual.isPresent(), equalTo(true));
+    FilmInfoDto dto = actual.get();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "actual.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)